Makefile

```diff
@@ -1,13 +1,13 @@
+REGISTRY ?= quay.io
+DEFAULT_TAG = latest
+
+DOCKERFILE := Dockerfile
 ifeq ($(TARGET),rhel)
-  DOCKERFILE := Dockerfile.rhel
-  REPOSITORY := openshiftio/rhel-fabric8-analytics-f8a-bq-manifests-job
+  REPOSITORY ?= openshiftio/rhel-fabric8-analytics-f8a-bq-manifests-job
```

Comment:
Need just one image if we are using UBI.

Reply:
I referred changes done on API server, it was having this changes. Do you want me to avoid RHEL image now?

Reply:
Yes, you can remove it.